modules/nw-dual-stack-convert.adoc

```diff
@@ -9,6 +9,11 @@ As a cluster administrator, you can convert your single-stack cluster network to
 After converting to dual-stack networking only newly created pods are assigned IPv6 addresses. Any pods created before the conversion must be recreated to receive an IPv6 address.
 ====
 
+[IMPORTANT]
+====
+Before proceeding make sure your OpenShift cluster uses at least version 4.12.5. Otherwise it may be broken due to the bug https://issues.redhat.com/browse/OCPBUGS-6040.
```

Comment:
```suggestion
Before proceeding, make sure your OpenShift cluster uses version 4.12.5 or later. Otherwise, the conversion can fail due to the bug link:https://issues.redhat.com/browse/OCPBUGS-6040[ovnkube node pod crashed after converting to a dual-stack cluster network].
```